config/config.yml.example

```diff
@@ -22,6 +22,11 @@ development: &default
   # '(https?://.*\.?(meedan.com|meedan.org))'
   allowed_origins: '.*'
 
+  # Alegre Similarity Thresholds
+  #
+  image_similarity_threshold: 0.7
+  text_similarity_threshold: 0.7
+
```

Comment:
Please mark whether those configuration keys are mandatory or optional and what they are used for, like we do with other configuration keys.